SoftRenderingApp3D.App/DataStructures/ObjectSerializer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using SoftRenderingApp3D.App.DataStructures;
+using System.Text.Json;
+using System.IO;
+
+
+namespace SoftRenderingApp3D.App.DataStructures  {
+    public class ObjectSerializer
+    {
+        public void SerializeObjectToFile(DisplayModelData model, string filePath)
+        {
+            string jsonString = JsonSerializer.Serialize(model, new JsonSerializerOptions { WriteIndented = true });
+            File.WriteAllText(filePath, jsonString);
+        }
+    
+        public DisplayModelJsonData DeserializeObjectFromFile(string filePath)
```

Comment:
Deserialize should be a generic method like the Deserializer method that is called:
`public T DeserializeObjectFromFile<T>(string filePath)`